### ISSUE TITLE
Add Dropdown Suggestion

### ISSUE THREAD

**Issue by rakeen** [zeit/hyper#857]
Although we can go through available options with `tab`, it'd be nice to have a dropdown suggestion. That way users can see many options all at once.
Like Atom has this emoji suggestion feature...
![image](https://cloud.githubusercontent.com/assets/10359412/19277693/6792e6aa-8ffc-11e6-81a3-34aff1defa20.png)

**Comment by CodeTheory** [comment]
This is also a case for a plugin. Hyper.app needs to match up to current terminal standards in other areas before we can add features like this. 😄

**Comment by flybayer** [comment]
For anyone wanting to implement this, you should check out https://github.com/shockone/black-screen

**Comment by kakato10** [comment]
Hyper looks polished now, so maybe you can take a look at this request again? 
Further black-screen (last known as upterm) is now deprecated and Hyper seems as our only chance for this feature.
